morango/management/commands/cleanupsyncs.py

```diff
@@ -97,8 +98,7 @@ def handle(self, *args, **options):
         transfer_count = transfer_sessions.count()
 
         # loop over the stale sessions one by one to close them out
-        for i in range(transfer_count):
-            transfer_session = transfer_sessions[0]
```

Comment:
Oh, so this and below was only ever affecting the first transfer/sync session!

Reply:
No I think this was operating as expected. The sync session one though had the problem.